### ISSUE TITLE
training without warmup produces poor results and highly difficult to train

### ISSUE THREAD

**Issue by TaekyungKi** [TaekyungKi/SRGAN-pytorch#1]
Because of the GPU memory shortage in Colab environment, I tried to train with batch size for 8, which is half of the original batch size.
So I tried several times to train with small learning rate for both the generator and the discriminator, with respect to original setting.
I encountered several issues:
- I trained model without warmup stage, the model can not produce high quality images, even it is worse than the bicubic and nearest. Also the models are too sensitive to learning rate so the training is terriblely difficult.
- 
-
